tests/Collector/EventListener/SchemaFilterListenerTest.php

```diff
@@ -6,26 +6,26 @@
 
 use Doctrine\Bundle\MigrationsBundle\EventListener\SchemaFilterListener;
 use Doctrine\DBAL\Schema\Table;
-use Doctrine\Migrations\Tools\Console\Command\DoctrineCommand;
+use Doctrine\ORM\Tools\Console\Command\ValidateSchemaCommand;
+use Doctrine\ORM\Tools\Console\EntityManagerProvider;
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Console\Event\ConsoleCommandEvent;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Output\NullOutput;
 
 class SchemaFilterListenerTest extends TestCase
 {
-    public function testItFiltersOutMigrationMetadataTableByDefault(): void
+    public function testItFiltersNothingByDefault(): void
     {
         $listener = new SchemaFilterListener('doctrine_migration_versions');
-
-        self::assertFalse($listener(new Table('doctrine_migration_versions')));
+        self::assertTrue($listener(new Table('doctrine_migration_versions')));
         self::assertTrue($listener(new Table('some_other_table')));
     }
 
-    public function testItDisablesItselfWhenTheCurrentCommandIsAMigrationsCommand(): void
+    public function testItFiltersOutMigrationMetadataTableWhenRunningSpecificCommands(): void
     {
         $listener          = new SchemaFilterListener('doctrine_migration_versions');
-        $migrationsCommand = new class extends DoctrineCommand {
+        $migrationsCommand = new class ($this->createStub(EntityManagerProvider::class)) extends ValidateSchemaCommand {
         };
 
```

Comment:
Isn't this still a valid test as the opposite for `testItFiltersOutMigrationMetadataTableWhenRunningSpecificCommands()`?

Reply:
The name is no longer valid, since the filter no longer disables itself (it's disabled by default and stays so until enabled).

There is another test that tests that it's disabled by default.

Reply:
It isn't about the name but about the 2 cases where we have an instance of UpdateCommand or ValidateSchemaCommand and not having an instance of ValidateSchemaCommand or UpdateCommand. `onConsoleCommand` is currently only covering one of those cases. UpdateCommand isn't covered either.

Reply:
Addressed, I think.